This patch disables 1280x768 and 1280x720 video modes because it behaves
in interesting manner with linux vesa driver, see #654823.

  -- mjt@tls.msk.ru

--- a/vbetables-gen.c	2011-01-31 20:27:54.000000000 +0300
+++ b/vbetables-gen.c	2012-03-05 00:38:01.667283519 +0400
@@ -57,5 +57,5 @@
 { 1152, 864, 32                      , 0x14c},
-{ 1280, 768, 16                      , 0x175},
-{ 1280, 768, 24                      , 0x176},
-{ 1280, 768, 32                      , 0x177},
+//{ 1280, 768, 16                      , 0x175},
+//{ 1280, 768, 24                      , 0x176},
+//{ 1280, 768, 32                      , 0x177},
 { 1280, 800, 16                      , 0x178},
@@ -81,5 +81,5 @@
 { 2560, 1600, 32                     , 0x18c},
-{ 1280, 720, 16                      , 0x18d},
-{ 1280, 720, 24                      , 0x18e},
-{ 1280, 720, 32                      , 0x18f},
+//{ 1280, 720, 16                      , 0x18d},
+//{ 1280, 720, 24                      , 0x18e},
+//{ 1280, 720, 32                      , 0x18f},
 { 1920, 1080, 16                     , 0x190},
